### PR TITLE
add timer to header of locust

### PR DESCRIPTION
**Pull request by rizalfr**

### CODE REVIEW

**Comment by erlanggakrisnamukti** [comment]
Better not restart the count when it stopped, but just pause the timer. In the other hand you should restart the timer when it start a new test or ramp

**Comment by rizalfr** [comment]
check lagi boleh @erlanggakrisnamukti @Amadeea

**Comment by erlanggakrisnamukti** [comment]
gas

File: locust/templates/index.html [hunk 1]
```diff
@@ -295,25 +295,35 @@
...
 
+      function startAndReset(){
+        startClock();
+        resetClock();
+      }
       function startClock() {
       	if (clicked === false) {
       		clock = setInterval("stopWatch()", 1000);
-      		clicked = true;
+      		clicked = true;;
```
Comment (Amadeea) [inline locust/templates/index.html]: kelebihan titik koma

File: locust/templates/index.html [hunk 2]
```diff
@@ -295,25 +295,35 @@
...
         document.getElementById("seconds").innerHTML = ++sec%60;
         document.getElementById("minutes").innerHTML = parseInt(sec/60,10);
+        console.log(sec);
       }
       function stopClock() {
       	window.clearInterval(clock);
-      	sec = 0;
-      	document.getElementById("seconds").innerHTML = 0;
-        document.getElementById("minutes").innerHTML = 0;
+        console.log(clock);
```
Comment (Amadeea) [inline locust/templates/index.html]: hilangin please

**Pull request merged**